src/styles/components/_footer.scss

```diff
@@ -9,23 +9,17 @@ $breakpoint-footer-margin: 25em;
 
   // Content wrapper
   &__content {
-    min-height: 20.75rem; /* 332px/16 */
     display: flex;
     flex-direction: column;
-    justify-content: space-between;
-    align-items: center;
-    gap: 3.75rem; // 60px/16
+    gap: 2.25rem; // 60px/16
     text-align: center;
-    padding: 3.75rem 0; /* top&bottom: 60px/16; left&right: 0 */
-    font-size: map.get($font-sizes, "sm");
-    margin: auto;
-    line-height: 1.5rem; /* 24px/16 */
+    padding: 1.5rem 0; /* top&bottom: 60px/16px; left&right: 0 */
+    font-size: map.get($font-sizes, "xs");
     transition: 500ms color;
     letter-spacing: 0.02em; // 2% in Figma
 
-    @media (min-width: $breakpoint-large-screen-860) {
-      line-height: 1.0625rem; /* 17px/16 */
-      font-size: map.get($font-sizes, "sm");
+    @media (min-width: $breakpoint-scale-up) {
+      line-height: 1.0625rem; /* 17px/16px */
```

Comment:
Not sure if this breakpoint is needed... What do you think about removing it?

Reply:
I think it was to match the original design perfectly (it had a one pixel difference between mobile/desktop), but because we're already straying from that design I don't think it's necessary. Removed it in [1d2e20b](https://github.com/sparkbox/accessible-components/pull/125/commits/1d2e20b8bd8628ec1a2e618bc943b1875cd8cd8b), along with another minor media query.